src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -1531,7 +1557,7 @@ public List<Action> retrieveActions(
                 Connector.lookupScanCredentials((Item) owner, getApiUri(), credentialsId, repoOwner);
         GitHub hub = Connector.connect(getApiUri(), credentials);
         Connector.configureLocalRateLimitChecker(listener, hub);
-        boolean privateMode = determinePrivateMode(apiUri);
+        boolean privateMode = !enableAvatar || determinePrivateMode(apiUri);
```

Comment:
but you are still making a request to GH on line 1562?  how is this helping the issue where we are making too many calls?

Reply:
The issue as I understand it is not that we are making too many calls but that we are making too many __anonymous__ calls. `determinePrivateMode(apiUri)` makes an anonymous call to do it's thing while `GHUser u = hub.getUser(getRepoOwner());` is making it's call with the credentials set above.

Reply:
Correct, `determinePrivateMode(apiUri)` consumes an anonymous call. `hub.getUser(getRepoOwner());` makes an authenticated call.